src/App.js

```diff
@@ -1,11 +1,17 @@
 import "./App.css";
-
-
+import {Routes, Route, } from "react-router-dom";
+import {LandingRoute,Products,Cart} from './routes'
+import Mockman from "mockman-js"
 function App() {
   return (
-    <div className="App">
-      
-    </div>
+    <>  
+      <Routes>
+      <Route path="/" element={<LandingRoute/>}/>
+      <Route path="/products" element={<Products/>}/>
+      <Route path='/cart' element={<Cart/>}/>
+      <Route path="/mock" element={<Mockman />}/>
+      </Routes>
+    </>

```

Comment:
there is a problem in indentation so you can use prettier for that to solve this issue.